add maven workflow for github action

gartcimore opened the pull request.

gartcimore committed:
add maven workflow for github action

gartcimore committed:
add release through github action

gartcimore committed:
add release through github action

gartcimore committed:
deploy package to git hub registry through github action

gartcimore committed:
deploy package to git hub registry through github action

gartcimore committed:
deploy package to git hub registry through github action

jhberthemet reviewed (changes requested):
Great contribution, let's remove the deploy par and make sure the result is the build with dependencies.

gartcimore committed:
add maven workflow for github action
commented out the part for the deploy (for now)
set correct info regards original repository

gartcimore committed:
revert to SNAPSHOT version

gartcimore committed:
add new flow to publish when a tag is pushed

gartcimore committed:
Create publish.yml

gartcimore committed:
change workflow name to distinguish

gartcimore committed:
test

gartcimore committed:
fix

gartcimore committed:
set version as not snapshot to test publishing

gartcimore commented:
removed publish on every push
added another workflow file to publish when a tag is pushed

gartcimore committed:
with cache

gartcimore committed:
use cache
